prov/hostspool/executor.go

```diff
@@ -33,67 +34,111 @@ import (
 	"github.com/ystia/yorc/v3/tosca"
 )
 
+const (
+	// endpointCapabilityName is the name of a TOSCA capability endpoint
+	endpointCapabilityName = "endpoint"
+	// publicAddressAttributeName is the name of an instance public address attribute
+	publicAddressAttributeName = "public_address"
```

Comment:
Just thinking but shouldn't we add at least this one as exported constants in the tosca package?

Reply:
has yes, will do